project/Frontend.scala

```diff
@@ -189,11 +190,11 @@ object Frontend extends Build with Prototypes {
     adminJobs
   )
 
-  val preview = application("preview").dependsOn(withTests(common), standalone).settings(
+  val preview = application("preview").dependsOn(commonWithTests, standalone).settings(
     RoutesKeys.routesImport += "scala.language.reflectiveCalls"
   )
 
-  val trainingPreview = application("training-preview").dependsOn(withTests(common), standalone).settings(
```

Comment:
they are, all the other projects are using `commonWithTests`, defined like this `val commonWithTests = withTests(common)`. Just wanted to make consistent